homeassistant/components/zwave/__init__.py

```diff
@@ -186,6 +186,15 @@ def _value_name(value):
     return '{} {}'.format(node_name(value.node), value.label).strip()
 
 
+def zwave_network_key(value):
+    """Validate a 16 byte value for zwave network keys."""
+    regex = re.compile(r'(0x\w\w,\s?){15}0x\w\w')
+    if not regex.match(value):
+        raise vol.Invalid('Invalid Z-Wave network key')
+
+    return str(value).lower()
```

Comment:
We don't need to copy to string since we now validate as string before this validator.

We can use `vol.Lower` at the end of the `vol.All` group.